app/tasks/raster_tile_cache_assets/symbology.py

```diff
@@ -51,67 +51,44 @@ class SymbologyInfo(NamedTuple):
     function: SymbologyFuncType
 
 
-def generate_date_conf_calc_string() -> str:
+def date_conf_merge_calc() -> str:
     """Create the calc string for classic GLAD/RADD alerts."""
-    day = "(A - ((A >= 30000) * 10000) - ((A >= 20000) * 20000))"
-    confidence = "(1 * (A >= 30000))"  # 0 for low confidence, 1 for high
+    day = "(A >= 20000) * (A.data % 10000)"
+    confidence = "(A.data // 30000)"  # 0 for low confidence, 1 for high
 
     red = f"({day} / 255)"
     green = f"({day} % 255)"
-    blue = f"(({confidence} + 1) * 100 + B)"
+    blue = f"(A.data >= 20000) * (({confidence} + 1) * 100 + B.data)"
 
     return f"np.ma.array([{red}, {green}, {blue}])"
 
 
-def generate_8_bit_integrated_calc_string() -> str:
+def integrated_alert_merge_calc() -> str:
```

Comment:
This methods seems like mostly a copy of the normal date conf encoding function. Would it be better to break it up like this:

- Method to generate red/blue/green channels
- Method to generate alpha channel
- Method that calls RGB method, and also calls if alpha method if necessary, and combined them all into masked array

Reply:
Good point, deduplication done!